### ISSUE TITLE
training speed is low, is it right?

### ISSUE THREAD

**Issue by xiakj** [wmcnally/kapao#75]
![image](https://user-images.githubusercontent.com/15704302/185286590-49abc5e9-eaaa-44f9-b12d-338505693ff6.png)

**Comment by wmcnally** [comment]
Yes training speed is slow due to large input size. 20 minutes per epoch doesn't sound out of the ordinary but training speed depends what hardware, batch size, and model size.
